crates/rpc/Cargo.toml

```diff
@@ -23,17 +36,8 @@ parking_lot.workspace = true
 serde.workspace = true
 serde_json.workspace = true
 ssz_types.workspace = true
+tempfile = "3"
 thiserror.workspace = true
+tokio = { version = "1", features = ["rt", "macros", "sync", "time"] }
 tracing.workspace = true
 tree_hash.workspace = true
-
-#ream-dependencies
-ream-beacon-api-types.workspace = true
-ream-bls.workspace = true
-ream-consensus.workspace = true
-ream-fork-choice.workspace = true
-ream-light-client.workspace = true
-ream-network-spec.workspace = true
-ream-node.workspace = true
-ream-p2p.workspace = true
-ream-storage.workspace = true
```

Comment:
please make the dependencies match the order it was before, I think they broke during the rebase